source/declarations.tex

```diff
@@ -4021,7 +4021,7 @@
 An abbreviated function template is equivalent to
 a function template\iref{temp.fct}
 whose \grammarterm{template-parameter-list} includes
-one invented type \grammarterm{template-parameter}
+one invented \grammarterm{type-parameter}
```

Comment:
The removal of "type" here opens the possibility that the _type-parameter_ is a template template parameter for the unconstrained case.

Reply:
I am not sure I understand what you mean

Reply:
The grammar for _type-parameter_ covers both type template parameters and template template parameters.  The old wording clearly ruled out template template parameters due to the "type" prefix.  The new wording does not.  You could say "type _type-parameter_" or just leave the existing wording in this one place.

Reply:
OH. Right, that's something that should be done in the paper, not here.